docs/docs-beta/docs/guides/declarative-automation.md

```diff
@@ -1,4 +1,136 @@
+
+```python
+import dagster as dg
+
+# Monitor updates only from a specific asset group
+dg.AutomationCondition.any_deps_match(
+    dg.AutomationCondition.newly_updated()
+).allow(dg.AssetSelection.groups("metrics")) # Asset group to monitor
+
+
+# Ignore missing upstream partitions from a specific asset
+dg.AutomationCondition.any_deps_match(
+    dg.AutomationCondition.missing()
+).ignore(dg.AssetSelection.keys("taxi_trips")) # Asset to ignore
+```
+
+## Condition labels
+
+To make conditions easier to understand, you can attach labels to sub-conditions, which will then display in the Dagster UI.
+
+Arbitrary string labels can be attached to any node in the `AutomationCondition` tree by using the `with_label()` method, allowing you to describe the purpose of a specific sub-condition:
+
+```python
+import dagster as dg
+
+in_progress_or_failed_parents = dg.AutomationCondition.any_deps_match(
+    dg.AutomationCondition.in_progress() | dg.AutomationCondition.failed()
+).with_label("Any parents in progress or failed")
+```
+
+Then, when viewing evaluation results in the UI, the label will display next to the condition:
+
+[TODO]
+
+## Event and status-based conditions
+
+TODO: REWORK THIS SECTION
+
+In some cases, you may want to use statuses and events in your automation conditions:
+
+- **Statuses** are persistent states that are and will be true for some period of time. For example, the `AutomationCondition.missing()` condition will be true only if an asset partition has never been materialized or observed.
+- **Events** are transient and reflect something that may only be true for an instant. For example, the `AutomationCondition.newly_updated()` condition will be true only if an asset partition was materialized since the previous evaluation.
+
+Using the `<A>.since(<B>)` operator, you can create conditions that detect if one event has happened more recently than another. Think of this as converting two events to a status - in this case, `A has occurred more recently than B` - as this will stay true for some period of time. This operator becomes true whenever `<A>` is true, and will remain true until `<B>` is also true.
+
+Conversely, it can also be useful to convert statuses to events. For example, the default `eager()` condition ensures that Dagster only tries to materialize a missing asset partition once using the following sub-condition:
+
+```python
+from dagster import AutomationCondition
+
+AutomationCondition.missing().newly_true().since(
+    AutomationCondition.newly_requested() | AutomationCondition.newly_updated()
+)
+```
+
+By using the `<A>.newly_true()` operator, you can turn the status of _"being missing"_ into a single event, specifically the point in time where an asset partition entered the _missing_ state. From there, you can ensure that an asset is materialized only once in response to detecting a missing partition.
```

Comment:
This section is very confusing. I'm not really sure what the 'goal' is i am trying to achieve with this